.ci/macos-steps.yaml

```diff
@@ -7,12 +7,12 @@ steps:
 # Set python version.
 - task: UsePythonVersion@0
   inputs:
-    versionSpec: '$(python.version)'
+    versionSpec: '3.7'
```

Comment:
We should keep the parameter here and set the version in the matrix.

Reply:
Done.